website/docs/docs/get-started/privatelink/snowflake-privatelink.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Configuring Snowflake PrivateLink"
+id: snowflake-privatelink
+description: "Configuring PrivateLink for Snowflake"
+sidebar_label: "Snowflake PrivateLink"
+---
+
+## Configure the Snowflake PrivateLink with AWS or Azure
+
+To complete the PrivateLink connection between the dbt and your private networks, first contact dbt Labs support to obtain the AWS or Azure `resource ID`. 
+
+A support case must then be created with Snowflake so they can grant access to dbt’s account or subscription. Snowflake teams prefer that customers open these support cases directly, rather than dbt Labs acting on your behalf. Please refer to [this article](https://community.snowflake.com/s/article/HowtosetupPrivatelinktoSnowflakefromCloudServiceVendors) for more information. Provide them with your dbt Cloud account ID along with any other information requested in the article. 
```

Comment:
```suggestion
Then, you must open a support case with Snowflake so they can grant access to dbt’s account or subscription. Snowflake teams prefer that you open these support cases directly, rather than dbt Labs acting on your behalf. For more information, you can refer to [Snowflake's article about setting up](https://community.snowflake.com/s/article/HowtosetupPrivatelinktoSnowflakefromCloudServiceVendors). Provide them with your dbt Cloud account ID along with any other information requested in the article. 
```